test/unit/base.test.js

```diff
@@ -37,6 +38,20 @@ describe('CONVERT FUNCTION TESTS ', function() {
         expect(conversionResult.output[0].data).to.have.property('info');
         expect(conversionResult.output[0].data).to.have.property('item');
 
+        done();
+      });
+    });
+    it('Should generate collection with collapsing unnecessary folders ' +
+     specPath3, function(done) {
+      var openapi = fs.readFileSync(specPath3, 'utf8');
+      Converter.convert({ type: 'string', data: openapi }, { schemaFaker: true }, (err, conversionResult) => {
+        expect(err).to.be.null;
+        expect(conversionResult.result).to.equal(true);
```

Comment:
Should add a test for folder name too. Tests shouldn't pass because of coincidence - test for the actual change you've made. If `/p/a/b` created a folder called `/p/a/b`, and no other folder, check for both.

Here, you aren't really checking for anything - just that one collection is generated, which has info/item propertieS, which every valid collection has.